internal/tui/components/app/app.go

```diff
@@ -644,18 +713,56 @@ func (m Model) getCurrentPageCmd() tea.Cmd {
 				}
 				allPageRows = append(allPageRows, page.Row{Row: formatter.StripOSCommandSequences(formatter.StripANSI(row))})
 			}
-			return nomad.PageLoadedMsg{Page: nomad.ExecCompletePage, TableHeader: []string{"Exec Session Output"}, AllPageRows: allPageRows}
+			return nomad.PageLoadedMsg{
+				Page:        nomad.ExecCompletePage,
+				TableHeader: []string{"Exec Session Output"},
+				AllPageRows: allPageRows,
+			}
 		}
 	case nomad.AllocSpecPage:
 		return nomad.FetchAllocSpec(m.client, m.alloc.ID)
 	case nomad.LogsPage:
-		return nomad.FetchLogs(m.client, m.alloc, m.taskName, m.logType, m.config.Log.Offset, m.config.Log.Tail)
+		return nomad.FetchLogs(
+			m.client, m.alloc, m.taskName, m.logType, m.config.Log.Offset, m.config.Log.Tail)
```

Comment:
Maybe your editor is automatically doing this reformatting, but if you don't mind undoing the formatting-only changes before merging, that would be great!